app/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -350,7 +350,8 @@ def test_valid_results_data_parsed_and_stored(
                             ],
                             "default_metrics": [],
                         }
-                    }
+                    },
+                    "analysis_start_time": "2022-08-31T04:30:03+00:00",
```

Comment:
The logic is checking for cases where either of `analysis_start_time` or `timestamp` can be present or missing, should we be testing for the four combinations of those cases?  Maybe a parameterized test?